rskj-core/src/main/java/co/rsk/peg/BridgeEvents.java

```diff
@@ -7,43 +7,57 @@
 
     LOCK_BTC("lock_btc",
             new CallTransaction.Param[]{
-                    new CallTransaction.Param(true, "receiver", SolidityType.getType("address")),
+                    new CallTransaction.Param(true, Fields.RECEIVER, SolidityType.getType(SolidityType.ADDRESS)),
                     new CallTransaction.Param(false, "btcTxHash", SolidityType.getType("bytes32")),
```

Comment:
would you like to add a string constant for `bytes32` here too?

Reply:
Done